packages/map/lib/README.md

```diff
@@ -89,7 +89,7 @@ public ngOnInit(): void {
 - `zoomIn()`: Zoom in
 - `zoomOut()`: Zoom out
 - `toggleFullscreen()`: Toggle fullscreen
-- `locate()`: Start zooming to current location
+- `locate(zoomlevel)`: Start zooming to current location with the given zoomlevel
```

Comment:
'with an optional zoom level'
Don't forget to mention the possible values and default value of `19`.
Very small extra note: zoom level should be two words, so perhaps it's better to update the variable to `zoomLevel`.